inc/c_util/object_lifetime_tracker.h

```diff
@@ -35,10 +35,19 @@ MU_DEFINE_ENUM(OBJECT_LIFETIME_TRACKER_REGISTER_OBJECT_RESULT, OBJECT_LIFETIME_T
 
 MU_DEFINE_ENUM(OBJECT_LIFETIME_TRACKER_UNREGISTER_OBJECT_RESULT, OBJECT_LIFETIME_TRACKER_UNREGISTER_OBJECT_RESULT_VALUES);
 
+#define OBJECT_LIFETIME_TRACKER_ACT_RESULT_VALUES \
+    OBJECT_LIFETIME_TRACKER_ACT_OK, \
+    OBJECT_LIFETIME_TRACKER_ACT_ERROR, \
+    OBJECT_LIFETIME_TRACKER_ACT_NOT_FOUND, \
+    OBJECT_LIFETIME_TRACKER_ACT_KEY_NOT_FOUND
```

Comment:
>OBJECT_LIFETIME_TRACKER_ACT_KEY_NOT_FOUND [](http://example.com/codeflow?start=4&length=41)

404 not found in specs. Is this (`OBJECT_LIFETIME_TRACKER_ACT_KEY_NOT_FOUND`)used?

Reply:
Fixed.